### ISSUE TITLE
PDF Export Enhancer

### ISSUE THREAD

**Issue by bokonist** [notion-enhancer/repo#22]
**problem**

As the custom themes make the UI quite different from normal Notion users, there's no (easy) option to show your notion page to someone as you view it. Also, another feature missing from Notion is the inability to export to dark mode PDF.

**solution**

how will this feature appear/act?

Exporting to PDF will export the page as we see it (with the custom theme applied), to a PDF file.

how could this be technically implemented using the available
capabilities of the enhancer and notion?
Not sure, but I currently do it with Stylus (chrome extension), to style the website with the same css files as the theme i'm using, then rightclick+print, save as PDF without the headers+footers. It's as close as I can do this currently, as you can imagine, it won't be a 100% matching export as you can't style fonts with Stylus.

**cons**

Can't think of one, this is just to extract the data to an external file.

**alternatives**

would a different feature remove the need for this?
No.

does this need to be implemented by the enhancer? could an external service work better?
It's quite dependent on the notion enhancer theme, so I think it has to be an enhancer feature.

**Comment by dragonwocky** [comment]
no. the closest you can get at the moment is to collapse the sidebar and then use ctrl+p to print to pdf
